### ISSUE TITLE
Pending specs become failures after using `testSpecs`

### ISSUE THREAD

**Issue by PierreR** [mitchellwrosen/tasty-hspec#15]
I might be missing something but I have tried to wrap Spec with `testSpecs`. It does work but all pending specs are now printed as failures. Is this expected ?

**Comment by mitchellwrosen** [comment]
Definitely not! Can you please make a reproducible repo for me to investigate?

**Comment by mitchellwrosen** [comment]
Oh, I misread this bug report. Yes, hspec `pending` tests get translated as failures - there's no equivalent `Tasty` result. The only other option is to instead report pending tests as successes.

**Comment by ChShersh** [comment]
@mitchellwrosen How can I report pending tests as successes? I have very simple pending test:

```haskell
spec_exampleSpec :: Spec
spec_exampleSpec = describe "example" $
    xit "pending example" $
        True `shouldBe` True
```

And I see that this tests fails with `tasty-hspec` while with `hspec` I see `Pending` output. What should I do in order to not make this test executed but also not result in failure with `tasty-hspec`?

**Comment by mitchellwrosen** [comment]
@ChShersh I've added a `--treat-pending-as` flag (and associated `TreatPendingAs` option), published as `tasty-hspec-1.1.5`.

**Comment by ChShersh** [comment]
@mitchellwrosen Thanks a lot! I will give it a try 👍
